src/country.spec.ts

```diff
@@ -51,16 +52,16 @@ describe('country', () => {
 
     assert.throws(
       () => country(at).getCountry(undefined as unknown as CountryAlpha2),
-      /^TypeError: Country not found for code 'undefined'$/u,
+      /^TypeError: Country not found for code 'undefined'$/u, // This is to check for the Country not found as undefined error
```

Comment:
this seems to be an exact match, so maybe just use plain string instead of regex?